src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandBase.cs

```diff
@@ -123,17 +123,18 @@ protected bool VerifySignatures
         /// <param name="parseResult">The results of parsing the command line.</param>
         /// <returns><see langword="true"/> if signatures of packages and installers should be verified.</returns>
         /// <exception cref="GracefulException" />
-        private static bool ShouldVerifySignatures(ParseResult parseResult)
+        private static bool ShouldVerifySignatures(ParseResult parseResult) =>
+            ShouldVerifySignatures(parseResult.GetValue(WorkloadInstallCommandParser.SkipSignCheckOption));
+
+        public static bool ShouldVerifySignatures(bool skipSignCheck = false)
         {
             if (!SignCheck.IsDotNetSigned())
             {
                 // Can't enforce anything if we already allowed an unsigned dotnet to be installed.
                 return false;
             }
 
-            bool skipSignCheck = parseResult.GetValue(WorkloadInstallCommandParser.SkipSignCheckOption);
```

Comment:
This value was unused.

Reply:
how is this unused? It's used on line 137

Reply:
I think I flubbed in my comment here. I was likely thinking of something else I reverted. The value is used, but passed in now as a parameter. Line 129 and then the same assignment here in line 127.